docs/modules/ROOT/pages/index.adoc

```diff
@@ -2,6 +2,8 @@
 
 A library of smart contracts that use ciphertext for amount, allowing for a wide variety of confidential use-cases, such as confidential tokens, auctions, vesting, voting etc. While the contracts are not written in an opinionated method (other than using the standard encrypted values published by Zama), for testing and examples in the documentation, the https://github.com/zama-ai/fhevm-solidity[Zama fhEVM] will be used to operate on and decrypt https://www.zama.ai/introduction-to-homomorphic-encryption[FHE] ciphertext.
 
+NOTE: All contracts must set their respective co-processor configuration during construction (or initialization). This can be done automatically by inheriting contracts in https://github.com/zama-ai/fhevm/blob/e376d87f5c0699a2611ab4933b4767490c91f1cd/library-solidity/config/ZamaConfig.sol#L54-L73[`ZamaConfig`].
```

Comment:
Very minor, maybe that for doc maintenance purposes:
```suggestion
NOTE: All contracts must set their respective co-processor configuration during construction (or initialization). This can be done automatically by inheriting contracts in https://github.com/zama-ai/fhevm/blob/v0.7.12/library-solidity/config/ZamaConfig.sol#L47-L73[`ZamaConfig`].
```